luigi/parameter.py

```diff
@@ -917,7 +928,7 @@ def normalize(self, x):
         :param str x: the value to parse.
         :return: the normalized (hashable/immutable) value.
         """
-        return tuple(x)
+        return _recursively_freeze(x)
```

Comment:
[The documentation says `<JSON string>`](https://github.com/spotify/luigi/blob/master/luigi/parameter.py#L905)